### ISSUE TITLE
Recoil does not return weapon to previous aiming location

### ISSUE THREAD

**Issue by Silverfang22** [JKGDevs/JediKnightGalaxies#219]
There should be new fields in the .wpn files of how long it takes to aim to new aiming height and how long it takes to return to original aiming location

Possibly different for horizontal and vertical

**Comment by DarthFutuza** [comment]
Are you sure it should?  In my opinion, recoil should force the user to have to reorient their crosshairs to where they're wanting to aim, requires more work and skill I think in compensating for recoil.  Maybe I'm misunderstanding what you want though.

**Comment by ensiform** [comment]
Well to me @DarthFutuza, that sounds like adding artificial "skill" for the sake of it rather than making something that everyone can enjoy.  Most people are used to recoil being some sort of temporary thing especially if you stop firing more or less?

**Comment by DarthFutuza** [comment]
@ensiform well it stops moving your cross hairs once you stop firing, it just doesn't automatically revert the cross hair location to the original position.  I find that both unnatural and intuitive when your cursor/cross hair moves on its own when taking no action.  Plus the current mechanic has been in JKG for 2+ years so changing it now is a big change to the way the game works.

**Comment by ensiform** [comment]
But if you want to draw in crowds of more than the 3 players you have now...

**Comment by Pande** [comment]
Yeah have a look at how call of duty returns the sights. It's very
intuitive.

**Comment by Silverfang22** [comment]
Besides, you can compensate for the recoil before it returns if you're rapidly firing, then when you stop firing you have to compensate for the weapon suddenly not recoiling anymore, it's more realistic in the end.

**Comment by Pande** [comment]
I can't recall but perhaps an exponential 'snap' to the old yaw angle the
closer you put the gun back to it would alleviate the post firing 'anti
recoil' but tbh I don't think it's a big deal and I'm tempted to agree with
dang about the realism. I gotta go play that cod game or something similar
to confirm, but I do remember that it was particularly fluid and felt right.
